src/features/iam/iamHelpers.ts

```diff
@@ -22,6 +23,16 @@ async function askForProjectId(): Promise<string | undefined> {
   return project?.projectId
 }
 
+async function enterWallet(): Promise<string | undefined> {
```

Comment:
Let's use consistent naming:
```suggestion
async function askForWalletUrl(): Promise<string | undefined> {
```

Reply:
Renamed